grc/gui/Application.py

```diff
@@ -811,11 +834,16 @@ def flow_graph_update(fg=flow_graph):
 
         selected_blocks = list(flow_graph.selected_blocks())
         selected_block = selected_blocks[0] if selected_blocks else None
+        # See if a connection has modifiable parameters or grey out the entry
+        # in the menu
+        selected_connections = list(flow_graph.selected_connections())
+        selected_connection = selected_connections[0] if selected_connections else None
+        selected_conn_has_params = selected_connection and bool(len(selected_connection.params))
```

Comment:
IDEA: unless we know, that the clicked connection is at index 0, I suggest to disable the properties. Otherwise a use can't know which one is being edited (from e.g. the Dialog title)

Reply:
Ah yeah, good point. If you've selected multiple connections.